CHANGELOG.md

```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.18.8
+
+_Jan 11, 2024_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- ⏰ Support date-fns v3 (#11659) @LukasTy
+- 🌍 Add missing Danish (da-DK) locale export (#11641) @etlos
```

Comment:
Should probably add the component lines that are impacted (data grid ? pickers ?)

Reply:
```suggestion
```

I taught it was about adding a locale, but it just a forgotten export fix